osu.Game/Overlays/Rankings/Tables/PerformanceTable.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using System.Collections.Generic;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Users.Drawables;
+using osuTK;
+using osu.Game.Online.API.Requests.Responses;
+
+namespace osu.Game.Overlays.Rankings.Tables
+{
+    public class PerformanceTable : RankingsTable<APIUserRankings>
+    {
+        public PerformanceTable(int page = 1)
+            : base(page)
+        {
+        }
+
+        protected override TableColumn[] CreateHeaders() => new[]
+        {
+            new TableColumn("", Anchor.Centre, new Dimension(GridSizeMode.Absolute, 50)), // place
```

Comment:
if you are repeating the same code in three files, you are doing it wrong. the shared portion should be shared.